src/sentry/api/endpoints/project_repo_path_parsing.py

```diff
@@ -130,6 +134,12 @@ def post(self, request: Request, project) -> Response:
         source_url = data["source_url"]
         stack_path = data["stack_path"]
 
+        # If stack_path uses backslashes, convert them to forward slashes for find_root function
+        is_backslash_stack_path = False
+        if "\\" in stack_path:
+            is_backslash_stack_path = True
+            stack_path = stack_path.replace("\\", "/")
+
```

Comment:
This works, however it can be done, IMO cleaner, using [pathlib](https://docs.python.org/3/library/pathlib.html)

```python
>>>  PureWindowsPath("C:\\sentry\\project_stacktrace_link.py").as_posix()
'C:/sentry/project_stacktrace_link.py'

>>> PureWindowsPath("/sentry/project_stacktrace_link.py").as_posix()
'/sentry/project_stacktrace_link.py'
```